rfc/12/README.md

```diff
@@ -81,4 +81,4 @@ There are 2 options for health checks integration into Pacemaker:
 
 ## See also
 
-* [Original list of LDR1 HA health checks. The spreadsheet has a lot of useful information](https://docs.google.com/spreadsheets/d/1xASlPnlFx1OmhKttbgOweHbmfbXMI6n3Pmp2TSO8ou8/edit#gid=1305517710)
+* [Original list of LDR1 HA health checks. The spreadsheet has a lot of useful information](https://seagatetechnology.sharepoint.com/:x:/r/sites/gteamdrv1/tdrive1224/LDR%20R1/HA/HA%20requirements%202020-01-14%20.xlsx?d=wdf76a255f0b84af5a771744ef00440d5&csf=1&web=1&e=gKXtZM)
```

Comment:
I just checked :
These sharepoint links are not accessible from outside seagate devices.
Either sharing settings need to be changed or the docs are needed to be converted to publically accessible formats (github or confluence docs)

Reply:
@suvratjoshi, this is a different problem to solve. we need check if the linked documents must be made available to the community or not. If they are planning or estimation related documents then I don't think community needs to have access to it and they are internal. Let's hold on to it until it is required by the community.

Reply:
Right @mssawant. Agree.
That is what even I was wondering.
For community those will not be needed mostly. But then in that case the link becomes invalid for community (as it shows Seagate login screen). If link still remains invalid for community, then we are still not fixing links in a way, fully.
Somebody will again file a github issue for that.

Should we 
1. remove these document refs from github pages and if required we can point those docs from our internal confluence pages where ever required, 
OR
2. just copy the documents to Hare sharepoint folders and no pointers to them for now from anywhere.

I think 2nd option is better.

Reply:
Agree with @suvratjoshi . @supriyachavan4398 pleaes remove "See also" section as per our discussion so we get unblocked for this PR and task. cc @mssawant @SunilSavanurSeagate

Reply:
Addressed.